query_optimizer/rules/AttachLIPFilters.cpp

```diff
@@ -128,7 +129,7 @@ void AttachLIPFilters::attachLIPFilters(
           lip_filter_configuration_->addBuildInfo(
               P::SingleIdentityHashFilterBuildInfo::Create(
                   pair.second->source_attribute,
-                  pair.second->estimated_cardinality * 8),
+                  std::max(64uL, pair.second->estimated_cardinality * 8u)),
```

Comment:
It seems that we won't need to change this value. If there are scenarios later that the configuration of this value makes a difference, then we can add a flag for that.